samples/basic/fade_led/prj_nrf52_pca10040.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_PRINTK=y
```

Comment:
This file is no longer needed. All these settings will get such values also without it.

Reply:
Part of demo how to use SW-PWM.
Removed.